zospy/analyses/mtf.py

```diff
@@ -237,3 +237,125 @@ def fft_through_focus_mtf_fromcfg(
     )
 
     return analysis.complete(oncomplete, result)
+
+
+def huygens_mtf(
+    oss: OpticStudioSystem,
+    pupilsampling: str | int = "32x32",
```

Comment:
```suggestion
    pupil_sampling: str | int = "32x32",
```

This keeps the naming consistent with other analyses (check e.g. [`psf.huygens_psf`](https://github.com/MREYE-LUMC/ZOSPy/blob/8b0d401870c36541df7117ebb18cdb764026740b/zospy/analyses/psf.py#L14)).
Please also update all other occurrences (e.g. in the unit tests and test reference data specification).

Reply:
I have made the requested variable name changes and propagated them to the unit tests and test reference data specification.